<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Voltage_divider_calculator_QTClass</class>
 <widget class="QWidget" name="Voltage_divider_calculator_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voltage_divider_calculator_QT</string>
  </property>
  <widget class="QLabel" name="inputLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output Voltage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>450</width>
     <height>170</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Voltage_divider_calculator_QT.qrc"/>
 </resources>
 <connections/>
</ui>
